core/src/main/scala/org/apache/spark/sql/rapids/tool/util/EventUtils.scala

```diff
@@ -188,16 +189,18 @@ object EventUtils extends Logging {
     m
   }
 
-  lazy val getEventFromJsonMethod:
-    String => Option[org.apache.spark.scheduler.SparkListenerEvent] = {
+  lazy val getEventFromJsonMethod: String =>
```

Comment:
why do we parse it like this with an Either vs just processing it as a regular event?  For instance we have the doOtherEvent where we could change classname to a string and compare it.

Reply:
brought this offline, will update later